poms/bom/pom.xml

```diff
@@ -327,6 +332,11 @@
                 <artifactId>camel-quarkus-direct</artifactId>
                 <version>${camel-quarkus.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.camel.quarkus</groupId>
+                <artifactId>camel-quarkus-fhir</artifactId>
+                <version>${project.version}</version>
```

Comment:
Please use `${camel-quarkus.version}` instead of `${project.version}` so that the end users are able to use the BOM as parent.

Reply:
Done